src/tests/utilsTest/render.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+import { render as rtlRender, RenderOptions } from '@testing-library/react'
+import { ChakraProvider } from '@chakra-ui/react'
+import { useConfigMock } from './mocks'
+
+Object.defineProperty(window, 'matchMedia', {
```

Comment:
@asacristani would you mind referencing this discussion here: https://github.com/chakra-ui/chakra-ui/discussions/6664

Reply:
@diversemix where? in the code?